Profile/EducationProfile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MarsQA_1.Helpers;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System.Threading;
+
+namespace MarsQA_1.SpecflowPages.Pages
+{
+    public class EducationProfile
+    {
+        public string Country { get; set; }
+        public string University { get; set; }
+        public string Title { get; set; }
+        public string Degree { get; set; }
+        public int YearOfGraduation { get; set; }
+        public string EducationMessage { get; set; }
+
+        IWebElement EducationTab => Driver.driver.FindElement(By.XPath("(//A[text()='Education'])[1]"));

```

Comment:
Do you need to access these elements from outside this class? if not, please add the private access modifier

Reply:
Yes, This is used in all classes in steps folder.